packages/nitro-protocol/addresses.json

```diff
@@ -0,0 +1,1139 @@
+{
+  "4": {
+    "rinkeby": {
+      "name": "rinkeby",
+      "chainId": "4",
+      "contracts": {
+        "NitroAdjudicator": {
+          "address": "0x94Fca2E2F57B25b2D9eA5d4FCEd6c5d8d9ad3172",
+          "abi": [
```

Comment:
This file is called `addresses.json`, but it contains more than addresses. Is that intentional?

Reply:
The most important information are the addresses, the ABI is just conveniently next to it... I think since the most important information are the addresses, `addresses.json` is reasonable

Reply:
This has the effect of growing the size of the git repo by a reasonable amount on each deploy. Considering the repo is already very large, if the intent is to simply list contract addresses, I think it should contain just the addresses.

Reply:
947c0b729